packages/core/base/src/services/crossmintModalService.ts

```diff
@@ -14,7 +14,7 @@ type MintQueryParams = {
     clientVersion: string;
     mintConfig: string;
     whPassThroughArgs?: string;
-    paymentMethod?: paymentMethods;
+    paymentMethod?: string;
```

Comment:
why this change?

Reply:
Becuase the type is `"fiat" | "ETH"` and we are technically passing `eth`